api/types/types.go

```diff
@@ -204,6 +204,7 @@ type Info struct {
 	RuncCommit         Commit
 	InitCommit         Commit
 	SecurityOptions    []string
+	Warnings           []string
```

Comment:
I kept this simple, and only return a list of strings; if we think it's important we can make the information more rich (return `level` and `message` for each warning, e.g. `level=info`, `level=warning`), but doing so felt like over-engineering, over-complicating.